.gitignore

```diff
@@ -0,0 +1,3 @@
+execution-environments/*/context
+execution-environments/*/ansible-navigator.log
+execution-environments/*/*artifact*.json
```

Comment:
Hmm, you need to rebase this PR so that changes that are already merged don't show up in its diff. Otherwise this gets extra hard to review.

Reply:
Fixed, thanks.